pkgs/by-name/ve/vep/package.nix

```diff
@@ -0,0 +1,132 @@
+# Instructions:http://www.ensembl.org/info/docs/api/api_installation.html
+# Do not use https://github.com/Ensembl/ensembl-vep/archive/release/${version}.zip
+# We cannot use INSTALL.pl but it’s not that bad:install the dependencies and copies the .pm files should be ok
+{ config
+, curl
+, lib
+, perlPackages
+, pkgs
+, stdenv
+, fetchFromGitHub
+, git
+, perl
+, unzip
+, makeWrapper
+, which
+, zlib
+}:
+
+with pkgs;
+
+let
+  customInstallPhase = ''
+    mkdir -p $out/${perl.libPrefix}/${perl.version}/
+    tests=$(find modules/ -mindepth 1 -maxdepth 1 -type d | grep -v t)
+    cp -r $tests $out/${perl.libPrefix}/${perl.version}/
+  '';
+
+  ensemblGit = n: r: s:
+    # Copy modules directly
+    stdenv.mkDerivation rec {
+      name = n;
+      version = "110";
+      # Neither tag nor release, so we need the commit...
+      src = fetchgit {
+        url = "https://github.com/Ensembl/${name}";
+        rev = r;
+        sha256 = s;
+      };
+      installPhase = customInstallPhase;
+    };
+
+  vepPlugins = fetchgit {
+    url = "https://github.com/Ensembl/VEP_plugins";
+    rev = "8f271c4848338dc7d504881ff71fdf2892c3d096";
+    sha256 = "sha256-LbaXwLFDP3m1QhRHwO9uh36BEFHE2NzL4xdxTb7/S5Q=";
+  };
+
+  # Install ensembl-xs, faster run using re-implementation in C of some of the Perl subroutines
+  ensembl-xs = perlPackages.buildPerlModule rec {
```

Comment:
Did you try `buildPerlPackage`, which I think would mean you don't have to override `buildPhase` and `installPhase`?